### PR TITLE
Add slackmojis extension

### PR DESCRIPTION
**Pull request by timoransky**
## Description

Simple extension which allows to search for gifs from [slackmojis.com](https://slackmojis.com). Users can copy the gif to clipboard or download it and upload to their Slack afterwards.

## Screencast
slackmojis-1
slackmojis-2

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/slackmojis/src/slackmojis-search.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        };
+      },
+      keepPreviousData: true,
+    },
+  );
+
+  return (
+    <Grid
+      columns={8}
+      inset={Grid.Inset.Medium}
```
Comment (pernielsentikaer) [inline extensions/slackmojis/src/slackmojis-search.tsx:38]: missing `isLoading` property

File: extensions/slackmojis/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Slackmojis
+
+an unofficial directory of the best custom Slack emojis from slackmojis.com
```
Comment (pernielsentikaer) [inline extensions/slackmojis/README.md:3]:
```suggestion
An unofficial directory of the best custom Slack emojis from slackmojis.com
```

File: extensions/slackmojis/package.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "slackmojis",
+  "title": "Slackmojis",
+  "description": "an unofficial directory of the best custom Slack emojis from slackmojis.com",
```
Comment (pernielsentikaer) [inline extensions/slackmojis/package.json:5]:
```suggestion
  "description": "An unofficial directory of the best custom Slack emojis from slackmojis.com",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by timoransky** [comment]
@pernielsentikaer thanks for the review, I implemented your suggestions, feel free to take a look

**Comment by pernielsentikaer** [comment]
Thanks for the fixes 🔥

**Pull request merged**